docs/zephyr-integration.md

```diff
@@ -0,0 +1,454 @@
+
+1. **Token Validation**: Verify your access token is valid and has appropriate permissions
+2. **Project Access**: Ensure the token has access to the specified project(s)
+3. **API Connectivity**: Test connection to the Zephyr Cloud API endpoint
+
+## Available Tools
+
+### Issue Coverage
+
+Tools for analyzing test coverage of JIRA issues and requirements.
+
+#### Get Issue Test Coverage
+- **Purpose**: Retrieve test cases that provide coverage for specific issues
+- **Use Cases**:
+  - Check which test cases cover a specific user story or bug fix
+  - Identify gaps in test coverage for requirements
+  - Generate coverage reports for stakeholders
+- **Parameters**:
+  - `issueKey` (required): JIRA issue key in format PROJECT-123
+  - `projectKey` (optional): Project key to scope the search
+- **Example**:
+  ```json
+  {
+    "issueKey": "PROJ-123",
+    "projectKey": "PROJ"
+  }
+  ```
+
+### Test Case Management
+
+Comprehensive CRUD operations for test case lifecycle management.
+
+#### Create Test Case
+- **Purpose**: Create new test cases with comprehensive metadata
+- **Parameters**:
+  - `name` (required): Test case name
+  - `projectKey` (required): Target project
+  - `folderId` (optional): Organization folder
+  - `priority` (optional): Priority level
+  - `status` (optional): Initial status
+- **Example**:
+  ```json
+  {
+    "name": "Test user login functionality",
+    "projectKey": "PROJ",
+    "folderId": 123,
+    "priority": "High",
+    "status": "Approved"
+  }
+  ```
+
+#### Update Test Case
+- **Purpose**: Modify existing test case properties
+- **Parameters**:
+  - `testCaseId` (required): ID of test case to update
+  - Updates can include name, description, priority, status, etc.
+
+#### Add Test Script
+- **Purpose**: Add executable test scripts to test cases
+- **Types**: STEP_BY_STEP, BDD, FREE_FORM
+- **Example**:
+  ```json
+  {
+    "testCaseId": 123,
+    "type": "STEP_BY_STEP",
+    "text": "1. Navigate to login page\n2. Enter credentials\n3. Click login"
+  }
+  ```
+
+#### Add Test Steps
+- **Purpose**: Add structured test steps with expected results
+- **Example**:
+  ```json
+  {
+    "testCaseId": 123,
+    "steps": [
+      {
+        "description": "Enter username",
+        "expectedResult": "Username field populated"
+      },
+      {
+        "description": "Enter password",
+        "expectedResult": "Password field populated"
+      }
+    ]
+  }
+  ```
+
+#### Link Test Case to Issue
+- **Purpose**: Establish traceability between test cases and JIRA issues
+- **Parameters**:
+  - `testCaseId` (required): Test case ID
+  - `issueKey` (required): JIRA issue key
+
+### Test Planning & Execution
+
+Tools for organizing and executing comprehensive test campaigns.
+
+#### Create Test Plan
+- **Purpose**: Organize test cases into executable plans
+- **Parameters**:
+  - `name` (required): Plan name
+  - `projectKey` (required): Target project
+  - `description` (optional): Plan description
```

Comment:
I think it would be better to stick to the names used in the different parts of our documentation and UI so I would use objective instead